manifests/kibana/basic-auth/deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kibana-basic-auth
+  namespace: logging
+  labels:
+    app: kibana
+    component: basic-auth
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: kibana
+        component: basic-auth
+    spec:
+      containers:
+      - name: caddy
+        image: dockermuenster/caddy:0.9.3
```

Comment:
Relying on https://hub.docker.com/r/dockermuenster/caddy/ seems a bit odd to me, as it meens introducing another party to trust. Maybe copy this to `giantswarm/caddy`?